src/samples/profile/v1/complete.json

```diff
@@ -6,52 +6,65 @@
     },
     "orcid": "0000-0002-1825-0097",
     "emailAddresses": [
-        "j.carberry@orcid.org"
+        {
+            "value": "j.carberry@orcid.org",
+            "access": "public"
+        },
+        {
+            "value": "j.carberry2@orcid.org",
```

Comment:
`j.carberry@restricted.example.com` or something that makes it clear it's private?